src/test/scala/za/co/absa/commons/scalatest/EnvFixtureSpec.scala

```diff
@@ -31,19 +33,25 @@ class EnvFixtureSpec extends AnyFlatSpec with Matchers with EnvFixture with Befo
     System.getenv.keySet()
   }
 
+  private val pathName = OperatingSystem.getCurrentOs match {
+    case OperatingSystems.WINDOWS => "Path" // this is what the variable actually by default looks like, at least on Win10
+    case _ => "PATH"
+  }
```

Comment:
For clarity reasons, let's make it a constant and call something like `AprioriExistingEnvVarName`

Reply:
Thanks, I applied your suggestion.